test/terra/backends/aer_simulator/test_options.py

```diff
@@ -165,13 +168,21 @@ def test_mps_approximation(self):
                 circuit.cx(0, i)
         circuit.save_statevector('sv')
 
-        result_exact = backend_exact.run(circuit, shots=shots).result()
-        sv_exact = result_exact.data(0)['sv']
+        result_swap_left = backend_swap_left.run(circuit, shots=shots).result()
+        sv_left = result_swap_left.data(0)['sv']
+        
+        result_swap_right = backend_swap_right.run(circuit, shots=shots).result()
+        sv_right = result_swap_right.data(0)['sv']
+        
         result_approx = backend_approx.run(circuit, shots=shots).result()
         sv_approx = result_approx.data(0)['sv']
-        # Check that the fidelity is reasonable
-        self.assertGreaterEqual(state_fidelity(sv_exact, sv_approx), 0.80)
+
+        # swap_left and swap_right should give the same state vector
+        self.assertAlmostEqual(state_fidelity(sv_left, sv_right), 1.0)
```

Comment:
It's strange to test this feature in a test called `test_mps_approximation`

Reply:
Would it be better if I named it `test_mps_options`? or do you think I should separate into 2 tests?

Reply:
I prefer two tests but `test_mps_options` is also ok